jicoco/src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -229,10 +224,7 @@ void start()
     private void initializeConnectAndJoin()
         throws Exception
     {
-        if (logger.isDebugEnabled())
-        {
-            logger.debug("Initializing a new MucClient for " + config);
-        }
+        logger.info("Initializing a new MucClient for " + config);
```

Comment:
Wouldn't that print some passwords in the logs?

Reply:
No, the password is not included in `toString`